msg/AudioNote.msg

```diff
@@ -0,0 +1,5 @@
+# Play a note (frequency) for a specific duration
```

Comment:
I think we should also add a `std_msgs/Header header` field.
Although we don't need it now, it may be useful in the future

Reply:
added in 7e3571b77922d018e722baf1b793eb1ca8bb362b